src/components/ConnectedList.jsx

```diff
@@ -68,9 +69,9 @@ class ConnectedList extends Component {
             <div>
               <h2>{t('connector.no-connectors-connected')}</h2>
               <p>{t('connector.get-info')}</p>
-              <NavLink to="/providers/all" className="col-button">
-                {t('connector.connect-account')}
-              </NavLink>
+              <a href={storeUrl} className="col-button" disabled={!storeUrl}>
+                <span>{t('connector.connect-account')}</span>
+              </a>
```

Comment:
Just a tip, there's a [`ButtonLink`](https://cozy.github.io/cozy-ui/react/#button) component in cozy-ui that does exactly this!